packages/server/tests/clients/servicesClient.ts

```diff
@@ -0,0 +1,246 @@
+        this.logger.info(`contract bytecode file: ${fileId?.toString()}`);
+
+        // Create the contract
+        const contractReceipt = await this.executeAndGetTransactionReceipt(new ContractCreateTransaction()
+            .setConstructorParameters(
+                new ContractFunctionParameters()
+            )
+            .setGas(75000)
+            .setInitialBalance(100)
+            .setBytecodeFileId(fileId || "")
+            .setAdminKey(this.client.operatorPublicKey || this.DEFAULT_KEY));
+
+        // The contract ID is located on the transaction receipt
+        const contractId = contractReceipt?.contractId;
+
+        this.logger.info(`new contract ID: ${contractId?.toString()}`);
+
+        return contractId;
+    };
+
+    async executeContractCall(contractId) {
+        // Call a method on a contract exists on Hedera, but is allowed to mutate the contract state
+        this.logger.info(`Execute contracts ${contractId}'s createChild method`);
+        const contractExecTransactionResponse =
+            await this.executeTransaction(new ContractExecuteTransaction()
+                .setContractId(contractId)
+                .setGas(75000)
+                .setFunction(
+                    'createChild',
+                    new ContractFunctionParameters()
+                        .addUint256(1)
+                ));
+
+        // @ts-ignore
+        const resp = await this.getRecordResponseDetails(contractExecTransactionResponse);
+        const contractExecuteTimestamp = resp.executedTimestamp;
+        const contractExecutedTransactionId = resp.executedTransactionId;
+
+        return { contractExecuteTimestamp, contractExecutedTransactionId };
+    };
+
+    async createAliasAccount(initialBalance = 1000) {
+        const privateKey = PrivateKey.generateECDSA();
+        const publicKey = privateKey.publicKey;
+        const aliasAccountId = publicKey.toAccountId(0, 0);
+
+        this.logger.trace(`New Eth compatible privateKey: ${privateKey}`);
+        this.logger.trace(`New Eth compatible publicKey: ${publicKey}`);
+        this.logger.debug(`New Eth compatible account ID: ${aliasAccountId.toString()}`);
+
+        const aliasCreationResponse = await this.executeTransaction(new TransferTransaction()
+            .addHbarTransfer(this._thisAccountId(), new Hbar(initialBalance).negated())
+            .addHbarTransfer(aliasAccountId, new Hbar(initialBalance)));
+
+        this.logger.debug(`Get ${aliasAccountId.toString()} receipt`);
+        await aliasCreationResponse?.getReceipt(this.client);
+
+        const balance = await this.executeQuery(new AccountBalanceQuery()
+            .setAccountId(aliasAccountId));
+        this.logger.info(`Balance of the new account: ${balance.toString()}`);
+
+        const accountInfo = await this.executeQuery(new AccountInfoQuery()
+            .setAccountId(aliasAccountId));
+        this.logger.info(`New account Info: ${accountInfo.accountId.toString()}`);
+        return {
```

Comment:
Can create an interface in here that this maps to and update the method signature so other classes know what to expect